lib/snapshot/runner.rb

```diff
@@ -18,6 +18,13 @@ def work
 
       clear_previous_screenshots if Snapshot.config[:clear_previous_screenshots]
 
+      Helper.log.info "Resetting all simulators screen scale to 1".green
+
+      all_device_types = `xcrun simctl list devicetypes`.scan(/.*\s\((.*)\)/)
```

Comment:
Use `FastlaneCore::Simulator.all` to get a list of all available simulators. 

```ruby
FastlaneCore::Simulator.all.each do |d|
end
```

Here are the available attributes: https://github.com/fastlane/fastlane_core/blob/master/lib/fastlane_core/simulator.rb#L55-L59